consensus/parlia/parlia.go

```diff
@@ -1538,11 +1585,30 @@ func (p *Parlia) getCurrentValidators(blockHash common.Hash, blockNum *big.Int)
 		return nil, nil, err
 	}
 
-	voteAddrmap := make(map[common.Address]*types.BLSPublicKey, len(valSet))
+	voteAddrMap := make(map[common.Address]*types.BLSPublicKey, len(valSet))
+	voteAddrExist := make(map[types.BLSPublicKey]bool, len(valSet))
+
+	// after FusionBlock, there may be same validator from BC and BSC
+	// we should remove duplicated consensus address and vote address
```

Comment:
but what you check now is only vote addr, but did not check consensus addr.  Is it possible that consensus addr can be duplicated?